lib/todo_lint/options.rb

```diff
@@ -0,0 +1,92 @@
+require "optparse"
+
+module TodoLint
+  # Handles option parsing for the command line application.
+  class Options
+    DEFAULT_REPORTER = ["Default", :stdout]
+
+    # Parses command line options into an options hash
+    # @api public
+    # @example Options.new.parse("todo_lint -c app.rb")
+    # @param args [Array<String>] arguments passed via the command line
+    # @return [Hash] parsed options
+    def parse(args)
+      @options = {}
+
+      OptionParser.new do |parser|
+        parser.banner = "Usage: todo_lint [options] [files]"
+
+        add_file_options parser
```

Comment:
ahh this is a clever way to get around the rubocop method length restriction. you may have noticed that I just disabled it for the CLI class. feel free to do the same here, or leave it